alws/routers/builds.py

```diff
@@ -50,4 +50,10 @@ async def restart_failed_build_items(build_id: int,
 
 @router.delete('/{build_id}/remove', status_code=204)
 async def remove_build(build_id: int, db: database.Session = Depends(get_db)):
-    return await crud.remove_build_job(db, build_id)
+    result = await crud.remove_build_job(db, build_id)
+    if not result:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
```

Comment:
I'd split up the situation where we do not have build and it's banned from release, because it's different statuses and explanations

Reply:
Fixed